scripts/test_docker_image.sh

```
@@ -51,7 +63,13 @@
 # Function to run a command inside the container
 run_in_container() {
   local cmd="$1"
-  docker run --rm -t "$FULL_IMAGE" /bin/sh -c "$cmd"
+  local env_vars="$2"
+  
+  if [ -z "$env_vars" ]; then
+    docker run --rm -t "$FULL_IMAGE" /bin/sh -c "$cmd"
+  else
+    docker run --rm -t $env_vars "$FULL_IMAGE" /bin/sh -c "$cmd"
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)